src/api/layer/bankTransactions.ts

```diff
@@ -120,15 +120,15 @@ export const listBankTransactionDocuments = get<{
   errors: unknown
 }>(
   ({ businessId, bankTransactionId }) =>
-    `/v1/businesses/${businessId}/bank-transactions/${bankTransactionId}/documents?content_disposition=INLINE`,
+    `/v1/businesses/${businessId}/bank-transactions/${bankTransactionId}/documents?content_disposition=ATTACHMENT`,
```

Comment:
I confirmed with @AndrewAcomb that the original behavior had no material impact in web browsers.